Dockerfile

```diff
@@ -28,6 +28,3 @@ COPY . ./
 EXPOSE 3000
 
 ENTRYPOINT ["bundle", "exec"]
-
-RUN rake db:create db:migrate  
```

Comment:
I was with problems to build the image, only removing this guy that I was able to buind and run the image. All the other changes that I've done was to run the image locally. with this configuration that I reverted I cant run the docker-compose on local because the database.yml dont find the docker-compose dtabase. If you are running it in this way maybe i'm doing something wrong.

Reply:
@AllanKlaus I haven't used this project via Docker for years, nor does it use Dockerfile on production. So you can ignore this for now. May I help you with turning tests green?

Reply:
now I understand. Yes if you can will help me a lot. this week I'll be overhead.

Reply:
@AllanKlaus, here you are: https://github.com/AllanKlaus/blog/pull/1

Reply:
nice guy. thanks :+1:

Reply:
Thank **you**, @AllanKlaus!